### PR TITLE
Add (unformatted) osso docs

### PR DESCRIPTION
**Pull request by sbauch**
Adds documentation for Osso per https://github.com/python-social-auth/social-core/pull/564

I'm having a lot of trouble figuring out how to lint/ autoformat these restructured text files. 

I see the Pep8 note in contributing doc, I guess that's been renamed to `pycodestyle`?

If someone could point me in the right direct for how to format from CLI or VSCode (im using venv) that would be super helpful. I'd be happy to update the contributing docs

### CODE REVIEW

File: docs/backends/osso.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+the user to the correct IDP. If you don't include one of these parameters, and instead implement a button, Osso 
+will display a hosted login page. Here's an example login form with ``email``:
+
+
+      <form action="{% url 'social:begin' 'osso' %}" method="post" class="login-form">
+          <label>Email</label>
+          <input type="email" name="email" />
+          {% csrf_token %}
+          <button type="submit">Sign in with SSO</button>
+      </form>
```
Comment (nijel) [inline docs/backends/osso.rst:42]: This should be a code block

```suggestion
.. code-block:: html+django

      <form action="{% url 'social:begin' 'osso' %}" method="post" class="login-form">
          <label>Email</label>
          <input type="email" name="email" />
          {% csrf_token %}
          <button type="submit">Sign in with SSO</button>
      </form>  
```

**Review by nijel — COMMENTED** [review]
Looks mostly good (see https://python-social-auth--79.org.readthedocs.build/en/79/backends/osso.html for rendered version).

**Comment by nijel** [comment]
Oh, and I don't think there is some auto-formatter for Sphinx used...

**Comment by sbauch** [comment]
Thanks! Was wondering why my template wasn't rendered as a code block, I've never seen this rst stuff before.

The formatting part that jumps out at me compared to other docs is line length, which is pretty annoying to do by hand, but I'm happy to if need be

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**